compiler/damlc/lib/DA/Cli/Damlc/Packaging.hs

```diff
@@ -534,28 +631,6 @@ exitWithError msg = do
     hFlush stderr
     exitFailure
 
-data DependencyInfo = DependencyInfo
```

Comment:
it was convenient that all the dependency checks took the same `DependencyInfo` argument, but I think it's better that they each take exactly the data they need, no more